tst/regression/CMakeLists.txt

```diff
@@ -42,11 +42,15 @@
 
 # Performance regression test
 list(APPEND TEST_DIRS advection_performance)
-list(APPEND TEST_PROCS ${NUM_MPI_PROC_TESTING})
+# Advection_performance should only be run on a single rank
+list(APPEND TEST_PROCS 1)
 list(APPEND TEST_ARGS "--driver ${PROJECT_BINARY_DIR}/example/advection/advection-example \
 --driver_input ${CMAKE_CURRENT_SOURCE_DIR}/test_suites/advection_performance/parthinput.advection_performance \
 --num_steps 5")
 list(APPEND EXTRA_TEST_LABELS "perf-reg")
+if ( NOT ${NUM_MPI_PROC_TESTING} EQUAL "1" ) 
+  message(STATUS "Note: advection_performance test will only run with a single rank.")
```

Comment:
Instead of outputting this every time you configure, would it make sense to instead rename the test to something like `advection_serial_performance`? Then it would be clear that the test is serial.

Reply:
I like that idea.